dali/test/python/test_RN50_data_pipeline.py

```diff
@@ -22,11 +22,29 @@
 import time
 
 class CommonPipeline(Pipeline):
-    def __init__(self, batch_size, num_threads, device_id, prefetch, fp16, nhwc):
+    def __init__(self, batch_size, num_threads, device_id, prefetch, fp16, nhwc, decoder_type, cache_size, cache_threshold, cache_type, reader_queue_depth):
         super(CommonPipeline, self).__init__(batch_size, num_threads, device_id, prefetch_queue_depth=prefetch)
+        if decoder_type == 'roi':
+            print('nvjpeg roi')
```

Comment:
Maybe something like
```suggestion
            print('Using nvJPEG with ROI')
```

Reply:
will do